final_prep/mod_zero_hero.rb

```diff
@@ -1,52 +1,87 @@
 
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "Watch out villains, I'm #{hero_name}!"
+catchpharse = "Up, up and away!  I must #{special_ability} to the top of that building!"
+
+puts greeting
+puts catchpharse
+
 # Declare two variables - power AND energy - set to integers
 
+power = 70
+energy = 160
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 70
+full_energy = energy * 150
+
+puts full_power
+puts full_energy
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
-# Print the first sidekick to your terminal
+arch_enemies = ["The Joker", "Lex Luther", "Turbine"]
+sidekicks = ["Robin", "Supergirl", "Kid Flash" ]
 
+# Print the first sidekick to your terminal
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies.push("The Riddler")
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+puts arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.shift
 # Print the sidekicks array to terminal to ensure you added a new sidekick
+puts sidekicks
+
 
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
-
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
+def assess_situation(danger_level, announcement, excuse)
+  if danger_level >=50
+    puts excuse
+  elsif danger_level >10 && danger_level <50
+    puts announcement
+  else
+    puts "Meh. Hard pass."
+  end
+end
+
```

Comment:
Your if/else conditionals should be different based on the directions:
- "Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal"

Reply:
I've made the changes, are they correct?  Do i need to open another pull request after changes are made?